scholia/app/templates/base.html

```diff
@@ -665,7 +676,7 @@
       resolver: 'custom',
       events: {
         search: debounce((searchTerm, callback) => {
-          var url = "https://www.wikidata.org/w/api.php?callback=?";
+          var url = "https://www.wikdata.org/w/api.php?callback=?";
```

Comment:
There is a typo here: wikdata -> wikidata.

Reply:
oh good catch I changed the URLs for testing purposes